spec/mailers/notification_mailer_spec.rb

```diff
@@ -30,5 +30,27 @@
       expect(message.body.encoded).to include(work_activity.message)
       expect(message.body.encoded).to include("To view the notification, please browse to http://www.example.com/works/#{work.id}.")
     end
+
+    context "when the mesage has markdown" do
+      let(:work_activity) { FactoryBot.create(:work_activity, work: work, message: "I like to send [links](www.google.com)") }
+      it "generates the e-mail message" do
+        expect(message_delivery).to be_a(ActionMailer::Parameterized::MessageDelivery)
+        expect(message_delivery.message).to be_a(Mail::Message)
+        message = message_delivery.message
+        expect(message.to).to eq([user.email])
+        expect(message.from).to eq(["noreply@example.com"])
+        expect(message.subject).to eq("[pdc-describe] New Notification")
+        text_part = message.text_part
+        html_part = message.html_part
+        expect(text_part.content_type).to eq("text/plain; charset=UTF-8")
+        expect(html_part.content_type).to eq("text/html; charset=UTF-8")
+        expect(html_part.encoded).to include("Hello #{user.display_name},")
+        expect(html_part.encoded).to include("To view the notification, please browse to http://www.example.com/works/#{work.id}.")
+        expect(html_part.encoded).to include("I like to send &lt;a href=&quot;www.google.com&quot;&gt;links&lt;/a&gt;")
+        expect(text_part.encoded).to include(work_activity.message)
+        expect(text_part.encoded).to include("Hello #{user.display_name},")
+        expect(text_part.encoded).to include("To view the notification, please browse to http://www.example.com/works/#{work.id}.")
```

Comment:
Are we expecting the text to include "I like to send ..." markdown? Or are we expecting it not to include it? I'd suggest adding an assertion one way or the other, to make sure we're getting the behavior we're expecting.

Reply:
Yes, this is done here: https://github.com/pulibrary/pdc_describe/pull/875/files#diff-9860398629ad665126bc447cc0b151e236450de156e8bc64a313c18db62bc7c3R50